ontology/spdx-ontology.owl.xml

```diff
@@ -3146,15 +2867,6 @@ IMAGE if the file is assoicated with an picture image file (MIME type of image/*
     
 
 
-    <!-- http://spdx.org/rdf/terms#reviewed -->
-
-    <owl:NamedIndividual rdf:about="http://spdx.org/rdf/terms#reviewed"/>
-    <rdf:Description>
-        <owl:qualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:qualifiedCardinality>
-    </rdf:Description>
```

Comment:
Curious what this individual was removed - does not seem related to the rest of the PR

Reply:
Sorry, I forgot to mention this one in the PR: this seemed to be another pair of spurious redefinitions added by Protege. 
1) Elsewhere #reviewed is also defined as an ObjectProperty. And separately in the file there is a rdf:Description that sets its status to deprecated (ideally this would be in the same XML element). The text I deleted is a 3rd place in the file that further defines it as a NamedIndividual (which would cause undesirable OWL punning to have it as a NamedIndividual and a Property) 
2) the qualifiedCardinality makes no sense standalone like this. It's not part of the NamedIndividual element. Even if it were, qualifiedCardinality can only validly be used as a property of an owl:Restriction.

Reply:
Thanks for the explanation - I'm not exactly sure how this ended up this way, but it looks like something we should remove.